modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache

```diff
@@ -1043,6 +1050,13 @@ def get_anyof_instances(self, model_args, constant_args):
         return anyof_instances
 
     for anyof_class in self._composed_schemas()['anyOf']:
+        if anyof_class is none_type:
+            # The only way the input data can match the 'None' type
+            # is if the input data is the 'null' value.
+            if model_args is None:
+                # The input data matches the 'None' type.
+                oneof_instances.append(model_args)
```

Comment:
Did you mean anyof_instances.append..?
Also, I don't think that we should add model_args to anyof_instances.
anyof_instances is meant to store ModelSimple or ModelComposed instances, not None.
Let's just delete this if clause and continue.

Reply:
Yes, good catch, I copy/pasted.

Reply:
> Did you mean anyof_instances.append..?
> Also, I don't think that we should add model_args to anyof_instances.
> anyof_instances is meant to store ModelSimple or ModelComposed instances, not None.
> Let's just delete this if clause and continue.

But then potentially `if len(anyof_instances) == 0:` would be true, if the input data is the `null` value and the OAS schema has `anyOf: [type: 'null', a, b, c]`

Reply:
Yes, return None. The spec says that None is a valid value. We raise exceptions if deserialization fails.

Reply:
We never get to counting oneOf or anyOf instance because we returned None in the `ComposedSchema.__new__` method and `ComposedSchema.__init__` is never called.

Reply:
```
a = FruitReq(None)
a == None -> True
isisnstance(a, FruitReq) -> False
isisnstance(a, none_type) -> True
```

Reply:
done